.travis.yml

```diff
@@ -15,7 +15,7 @@ env:
     - USE_DEB=true  ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
 matrix:
   allow_failures:
-    - env: USE_DEB=true  ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - env: USE_DEB=true  ROS_DISTRO="melodic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
```

Comment:
Did you mean to edit the `allow_failures` OR should you have edited the matrix above?

Reply:
meant to edit both allow_failures for the shadow build and all matrix env to melodic. fixed and cleaned up useless comments.